nautobot_circuit_maintenance/views.py

```diff
@@ -34,7 +34,9 @@ def get_extra_context(self, request, instance):
         """Extend content of detailed view for Circuit Maintenance."""
         maintenance_note = models.Note.objects.filter(maintenance=instance)
         circuits = models.CircuitImpact.objects.filter(maintenance=instance)
-        parsednotification = models.ParsedNotification.objects.filter(maintenance=instance)
+        parsednotification = models.ParsedNotification.objects.filter(maintenance=instance).order_by(
+            "raw_notification__date"
+        )
```

Comment:
Same question here - `ParsedNotification` is natively ordered by its own `date` field, what's the intent behind ordering by `raw_notification__date` instead?

Reply:
Actually, looking at the screenshot you provided, would it make sense to reverse the ordering so that the newest notification appears at the top of the table?